### PR TITLE
chore(internal/actions/cmd): add nested commit output to updateall

### PR DESCRIPTION
**Pull request by quartzmo**
closes: #8617

### CODE REVIEW

**Comment by quartzmo** [comment]
@noahdietz Great suggestions, thanks for the edits!

File: internal/actions/cmd/updateall/main.go [hunk 1]
```diff
@@ -103,20 +119,30 @@ func dependson(mod, dep, version string) (bool, error) {
...
 		return err
 	}
 
+	if !*quiet {
+		scope := strings.TrimPrefix(strings.TrimSpace(modDir), rootDir+"/")
+		if scope == "" || scope == "main" ||  strings.Contains(scope, "internal/") {
+			return nil
+		}
+		nestedCommits.WriteString("BEGIN_NESTED_COMMIT\n")
+		nestedCommits.WriteString(fmt.Sprintf("%s(%s): %s\n",  commitLevel, scope, commitMsg))
```
Comment (noahdietz) [inline internal/actions/cmd/updateall/main.go:143]:
```suggestion
		nestedCommits.WriteString(fmt.Sprintf("%s(%s): %s\n", commitLevel, scope, commitMsg))
```

File: internal/actions/cmd/updateall/main.go [hunk 2]
```diff
@@ -103,20 +119,30 @@ func dependson(mod, dep, version string) (bool, error) {
...
-	c.Dir = mod
+	c.Dir = modDir
 	_, err = c.Output()
 	if err != nil {
 		return err
 	}
 
+	if !*quiet {
+		scope := strings.TrimPrefix(strings.TrimSpace(modDir), rootDir+"/")
+		if scope == "" || scope == "main" ||  strings.Contains(scope, "internal/") {
```
Comment (noahdietz) [inline internal/actions/cmd/updateall/main.go:139]:
```suggestion
		if scope == "" || scope == "main" || strings.Contains(scope, "internal/") {
```

**Review by noahdietz — APPROVED** [review]
Added suggestions for the vet warnings. I will commit them and merge this. Good job @quartzmo !

**Pull request merged**